nixos/modules/system/boot/kernel.nix

```diff
@@ -8,6 +8,14 @@ let
   inherit (config.boot.kernel) features randstructSeed;
   inherit (config.boot.kernelPackages) kernel;
 
+  modulesTypeDesc = ''
+    This can either be a list of modules, or an attrset. In an
+    attrset, names that are set to `true` represent modules that will
+    be included. Note that setting these names to `false` does not
+    prevent the module from being loaded. For that, use
+    {option}`boot.blacklistedKernelModules`.
+  '';
```

Comment:
Should it be `coercedTo (... true) (attrsOf (enum [ true false "auto" ]))` then?

Reply:
I don't understand. That isn't what these options do.